pkg/config/cert_rotation_config.go

```diff
@@ -21,6 +21,9 @@ import "github.com/hashicorp/go-multierror"
 type CertRotationConfig struct {
 	*CryptoConfig
 
+	// ProjectID is the Google Cloud project ID.
```

Comment:
This is a change throughout. I'm not sure where to note it, but this requires services to set a `PROJECT_ID` envvar instead of doing the gcpmetadata lookup. I _think_ I need to update the Terraform to inject these, but I wanted to get feedback before doing all that work.

Long story short, I deleted the wrong KMS keys because I had `gcloud` logged into a different project (which the gcpmetadata package picks up). I'd prefer if we fully owned the config.

Reply:
I think we've discussed this before. `PROJECT_ID` should be a universal env var for all of our services.